community/modules/remote-desktop/chrome-remote-desktop/variables.tf

```diff
@@ -62,8 +62,8 @@ variable "instance_image" {
     project = string
```

Comment:
Since we are no longer going to test Debian 11, WDYT of removing it from our supported images?

Reply:
Do you mean from a larger HPC-Toolkit perspective?  

It might be worth bringing in Debian 12 as supported, setting Debian 11 as deprecated for a few months, then removing it.  I couldn't say that Debian 12 won't break anything.

Reply:
No, just from this module. 

The description above still says deb 11 is supported. It does not mention deb 12. That should probably be updated. You could also update the ansible to only support deb 12. I am not sure if it is validated anywhere else.